### PR TITLE
feat(integrations): update docstrings of the Keras callbacks

### PR DESCRIPTION
**Pull request by ayulockin**
Fixes WB-NNNN
Fixes #NNNN

Description
-----------
The arguments sections of the reference documentation for the Keras callbacks don't render properly. This PR will likely fix that issue.

Testing
-------
How was this PR tested?

Checklist
-------
- [ ] Include reference to internal ticket "Fixes WB-NNNN" and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Ensure PR title compliance with the [conventional commits standards](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#conventional-commits)

### CODE REVIEW

**Comment by raubitsj** [comment]
@ayulockin can you fix up the conflicts and merge.

**Review by raubitsj — APPROVED** [review]
LGTM

File: wandb/integration/keras/callbacks/model_checkpoint.py [hunk 1]
```diff
@@ -46,28 +46,36 @@ class WandbModelCheckpoint(callbacks.ModelCheckpoint):
...
+            formatting options like `{epoch}` then `filepath` will be overwritten by
+            each new better model locally. The model logged as an artifact will still be
+            associated with the correct `monitor`.  Artifacts will be uploaded
+            continuously and versioned separately as a new best model is found.
+        save_weights_only: (bool) if True, then only the model's weights will be saved.
+        mode: (Mode) one of {'auto', 'min', 'max'}. For `val_acc`, this should be `max`,
             for `val_loss` this should be `min`, etc.
-        save_weights_only (bool): if True, then only the model's weights will be saved
-        save_freq (Union[SaveStrategy, int]): `epoch` or integer. When using `'epoch'`,
+        save_weights_only: (bool) if True, then only the model's weights will be saved
```
Comment (raubitsj) [inline wandb/integration/keras/callbacks/model_checkpoint.py:68]: Looks like this has a dup a few lines up.

**Review by raubitsj — CHANGES_REQUESTED** [review]
Please fix merge conflict also

**Comment by ayulockin** [comment]
fixing conflict.

**Review by kptkin — APPROVED** [review]

**Pull request merged**